integration/spdk/Dockerfile

```diff
@@ -15,3 +15,16 @@ RUN cd spdk && \
     ./configure --disable-tests --without-vhost --without-virtio \
                 --with-rdma --target-arch=${ARCH} && \
     make
+
```

Comment:
instead of `make` in builder let's run 
```
RUN cd spdk && ./rpmbuild/rpm.sh --without-uring --without-crypto --without-fio --with-raid5 --with-vhost --without-pmdk --without-rbd --with-rdma --with-shared --with-iscsi-initiator --without-vtune --without-isal
```
and then inside the run second stage
```
RUN dnf install -y /root/rpmbuild/RPMS/${ARCH}/spdk-*${TAG}-1.${ARCH}.rpm
```

Reply:
It turns out it's roughly 50MB bigger using RPMs:

```
integration_spdk-target        latest         540e56346460   7 seconds ago    262MB
integration_xpu-spdk           latest         540e56346460   7 seconds ago    262MB
```

Reply:
Agree, RPMs give us standards and dependencies but a bit less efficient...